packages/terra-notification-dialog/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-notification-dialog",
+  "main": "lib/NotificationDialog.js",
+  "version": "0.0.0",
+  "description": "`terra-notification-dialog` is a notification dialog component built over the `terra-abstract-modal`. It has the highest z-index of 9001. It is a common component to be used for confirmation/acceptance criteria style dialogs.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-notification-dialog",
+    "NotificationDialog",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react-intl": "^2.4.0",
+    "terra-abstract-modal": "^1.2.0",
+    "terra-app-delegate": "^2.1.0",
+    "terra-base": "^3.1.0",
+    "terra-button": "^2.5.0",
+    "terra-doc-template": "^1.2.0",
+    "terra-icon": "^2.6.0",
+    "terra-modal-manager": "^3.0.0"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rimraf lib",
+    "compile:build": "babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "test": "npm run test:jest && npm run test:wdio",
+    "test:jest": "jest ./tests/jest/* --config ../../jestconfig.js",
```

Comment:
Lets update this script to be jest --config ../../jestconfig.js. This was a windows compat update made in #198